src/ZendeskApi.Client/ZendeskApiClient.cs

```diff
@@ -52,6 +52,7 @@ public HttpClient CreateClient(string resource = null)
             var authorizationHeader = _options.GetAuthorizationHeader();
```

Comment:
Can you bump the version number please? 🙏

Reply:
thank you, pushed! 🙏